Makefile

```diff
@@ -295,6 +298,10 @@ test-check-containers:
 tidy: tidy-js
 	tools/tidy
 
+.PHONY: test-containers-compose
+test-containers-compose:
+	for i in $(wildcard container/**/docker-compose.yaml); do cd "$$(echo $$i | cut -d "/" -f-2)"; sudo docker-compose up -d || exit 1; cd ../../; done
```

Comment:
I would avoid the `cd ../../` by doing the `cd` in a subprocess:

`(cd ...; sudo ...)`